doc/conf.py

```diff
@@ -49,6 +49,10 @@
 
 # generate autosummary even if no references
 autosummary_generate = True
+autodoc_default_options = {'inherited-members': None}
+numpydoc_class_members_toctree = False
+numpydoc_attributes_as_param_list = True
+default_role = 'autolink'  # XXX silently allows bad syntax, someone should fix
```

Comment:
ignore these 4 lines unless you want to dig deep into how sphinx works. It's copied from MNE.